tests/packagedcode/data/buildpack/paketo-buildpacks/java-memory-assistant/expectedbuildpack.json

```diff
@@ -0,0 +1,112 @@
+          "primary_language": null,
+          "description": "A Cloud Native Buildpack that installs the Java Memory Assistant agent",
+          "release_date": null,
+          "parties": [],
+          "keywords": [
+            "agent"
+          ],
+          "homepage_url": "https://github.com/paketo-buildpacks/java-memory-assistant",
+          "download_url": null,
+          "size": null,
+          "sha1": null,
+          "md5": null,
+          "sha256": null,
+          "sha512": null,
+          "bug_tracking_url": null,
+          "code_view_url": null,
+          "vcs_url": null,
+          "copyright": null,
+          "holder": null,
+          "declared_license_expression": "apache-2.0",
+          "declared_license_expression_spdx": "Apache-2.0",
+          "license_detections": [
+            {
+              "license_expression": "apache-2.0",
+              "license_expression_spdx": "Apache-2.0",
+              "matches": [
+                {
+                  "license_expression": "apache-2.0",
+                  "license_expression_spdx": "Apache-2.0",
+                  "from_file": "buildpack.toml",
+                  "start_line": 1,
+                  "end_line": 1,
+                  "matcher": "2-aho",
+                  "score": 100.0,
+                  "matched_length": 3,
+                  "match_coverage": 100.0,
+                  "rule_relevance": 100,
+                  "rule_identifier": "spdx_license_id_apache-2.0_for_apache-2.0.RULE",
+                  "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/spdx_license_id_apache-2.0_for_apache-2.0.RULE",
+                  "matched_text": "type: Apache-2.0"
+                }
+              ],
+              "identifier": "apache_2_0-d66ab77d-a5cc-7104-e702-dc7df61fe9e8"
+            }
+          ],
+          "other_license_expression": null,
+          "other_license_expression_spdx": null,
+          "other_license_detections": [],
+          "extracted_license_statement": "type: Apache-2.0",
+          "notice_text": null,
+          "source_packages": [],
+          "file_references": [],
+          "is_private": false,
+          "is_virtual": false,
+          "extra_data": {
+            "api_version": "0.7",
+            "id": "paketo-buildpacks/java-memory-assistant",
+            "targets": [
+              {
+                "arch": "amd64",
+                "os": "linux"
+              },
+              {
+                "arch": "arm64",
+                "os": "linux"
+              }
+            ]
+          },
+          "dependencies": [
+            {
+              "purl": "pkg:generic/sap-java-memory-assistant@0.5.0?arch=amd64",
+              "extracted_requirement": null,
+              "scope": "runtime",
+              "is_runtime": true,
+              "is_optional": false,
+              "is_pinned": false,
+              "is_direct": true,
+              "resolved_package": {},
```

Comment:
There is a bit more information here about dependencies that we can store, since the dependencies are also resolved. This would be a PackageData mapping, with extra information about the dependencies, including the purl fields, license info and others. Please check the spec to see what other info can be specified here, and add tests accordingly with richer data as applicable.